lib/datadog/core/telemetry/client.rb

```diff
@@ -74,13 +88,52 @@ def client_configuration_change!(changes)
           @emitter.request('app-client-configuration-change', data: { changes: changes, origin: 'remote_config' })
         end
 
+        def add_count_metric(namespace, name, value, tags)
+          return if !@metrics_enabled || forked?
+
+          @metric_queue.add_metric(namespace, name, value, tags, Metric::Count)
+        end
+
+        def add_rate_metric(namespace, name, value, tags)
+          return if !@metrics_enabled || forked?
+
+          @metric_queue.add_metric(namespace, name, value, tags, Metric::Rate)
+        end
+
+        def add_gauge_metric(namespace, name, value, tags)
+          return if !@metrics_enabled || forked?
+
+          @metric_queue.add_metric(namespace, name, value, tags, Metric::Gauge)
+        end
+
+        def add_distribution_metric(namespace, name, value, tags)
```

Comment:
I think we can remove the `add_` prefix here. It sounds more "ruby" that way.
What do you think?